Firestore/Example/Tests/Model/FSTFieldValueTests.mm

```diff
@@ -460,9 +468,8 @@ - (void)testValueEquality {
     @[ FSTTestFieldValue(FSTTestGeoPoint(1, 0)) ],
     @[
       [FSTReferenceValue referenceValue:FSTTestDocKey(@"coll/doc1")
-                             databaseID:[FSTDatabaseID databaseIDWithProject:@"project"
-                                                                    database:kDefaultDatabaseID]],
-      FSTTestFieldValue(FSTTestRef(@"project", kDefaultDatabaseID, @"coll/doc1"))
+                             databaseID:DatabaseId("project", DatabaseId::kDefaultDatabaseId)],
```

Comment:
This `firebase::firestore::util::WrapNSStringNoCopy` everywhere really impedes readability.